libvirt/resource_libvirt_domain.go

```diff
@@ -224,17 +226,17 @@ func resourceLibvirtDomainCreate(d *schema.ResourceData, meta interface{}) error
 	}
 
 	if ignition, ok := d.GetOk("coreos_ignition"); ok {
-		//var fw_cfg []defCmd
-		var fw_cfg []libvirtxml.DomainQEMUCommandlineArg
+		//var fwCfg []defCmd
```

Comment:
This line can be removed. We don't use it.

Reply:
yp